### DISCUSSION TITLE
Metamask pops up everytime I click "deploy"

### DISCUSSION THREAD

**Discussion by sherryyw** [smartcontractkit/full-blockchain-solidity-course-py#1797] [Q&A] [answered]
Metamask pops up and asks me to pay gas everytime I click "deploy" in the second lesson in the video. I am currently at the beginning of SimpleFactory section. But I don't see MM does the same thing in the video. 

Anyone knows why? 
![image](https://user-images.githubusercontent.com/98007232/204716950-e6715cae-1fd9-4c07-b9e0-2ce2f74f842e.png)

**Comment by cromewar** [comment, marked as answer]
Hello @sherryyw 
It depends on which environment you are using: 

![image](https://user-images.githubusercontent.com/20199180/204812048-d4f67dff-6d59-45ba-bcdd-bcb873d1fd1b.png)

If you are using a metamask it'll ask for payment every time, you can use REMIX VM instead.

Cheers.

**Reply by sherryyw** [reply]
Thank you!
